pyetrade/order.py

```diff
@@ -10,13 +10,68 @@
 
 """
 
+import dateutil.parser
 import logging
 import jxmlease
+from nose.tools import *
+import pdb
 from requests_oauthlib import OAuth1Session
 
 LOGGER = logging.getLogger(__name__)
 
 
+# some constants
+CALL = "Call"
+PUT  = "Put"
+
+
+# resp_format: xml (default) or json
+# empty_json: either [] or {}, depends on the caller's semantics
+def get_request_result(req, resp_format, empty_json):

```

Comment:
Let's handle network call result all in this single function, instead of duplicate the handling after each network call.

Reply:
In a future revision/refactor I want to have a base class/mix in and this will make a great method for that.